### PR TITLE
Prepare for 1.0.0 release

### PR DESCRIPTION
**Pull request by NilashishC**
Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>

##### SUMMARY
- Update docs

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
nxos_*

### CODE REVIEW

**Comment by NilashishC** [comment]
recheck

File: plugins/modules/nxos_interface_ospf.py [hunk 1]
```diff
@@ -16,19 +16,15 @@
...
-    "supported_by": "network",
-}
-
 
-DOCUMENTATION = """module: nxos_interface_ospf
+DOCUMENTATION = """
+module: nxos_interface_ospf
 extends_documentation_fragment:
 - cisco.nxos.nxos
 short_description: Manages configuration of an OSPF interface instance.
```
Comment (justjais) [inline plugins/modules/nxos_interface_ospf.py:24]: @NilashishC I believe for short description update u had diff PR, correct?

Comment (NilashishC) [reply plugins/modules/nxos_interface_ospf.py:24]: Not sure which one you're taking about.

Comment (justjais) [reply plugins/modules/nxos_interface_ospf.py:24]: ignore it, I checked for resource modules and it’s as expected

**Pull request merged**